src/variants.ini

```diff
@@ -733,6 +927,399 @@ nMoveRule = 0
 nFoldValue = loss
 stalemateValue = loss
 
+[gorogoroplus:gorogoro]
+startFen = sgkgs/5/1ppp1/1PPP1/5/SGKGS[LNln] w 0 1
+lance = l
+shogiKnight = n
+promotedPieceType = l:g n:g
+
+[dragonfly:chess]
+maxFile = 7
+maxRank = 7
+pocketSize = 7
+castlingKingsideFile = f
+castlingQueensideFile = b
+pieceDrops = true
+capturesToHand = true
+promotionRank = 7
+promotionPieceTypes = nbr
+doubleStep = false
+dropNoDoubled = p
+dropNoDoubledCount = 0
+startFen = rbbknnr/ppppppp/7/7/7/PPPPPPP/RBBKNNR[] w - - 0 1
+
+[kamikazerooks:chess]
+extinctionValue = win
+extinctionPieceTypes = r
+
+[mounted]
+maxRank = 7
+maxFile = 7
+pieceDrops = true
+capturesToHand = true
+stalemateValue = loss
+nFoldValue = loss
+extinctionValue = loss
+extinctionPseudoRoyal = true
+extinctionPieceTypes = k
+flagPiece = k
+whiteFlag = *7
+blackFlag = *1
+centaur = k
+bishop = b
+rook = r
+cannon = c
+customPiece1 = a:mBcpB 
+customPiece2 = s:FsfW
+startFen = 1cbkra1/2s1s2/7/7/7/2S1S2/1CBKRA1 w - - 0 1
+
+[racingchess:chess]
+flagPiece = k
+whiteFlag = *8
+blackFlag = *1
+flagMove = true
+checking = false
+
+[shinobimirror:crazyhouse]
+commoner = c
+bers = d
+archbishop = j
+fers = m
+shogiKnight = h
+lance = l
+promotionRank = 7
+promotionPieceTypes = -
+promotedPieceType = p:c m:b h:n l:r
+mandatoryPiecePromotion = true
+stalemateValue = loss
+perpetualCheckIllegal = true
+startFen = lh1ck1hl/pppppppp/8/8/8/8/PPPPPPPP/LH1CK1HL[LHMMDJlhmmdj] w - 0 1
+capturesToHand = false
+whiteDropRegion = *1 *2 *3 *4
+blackDropRegion = *5 *6 *7 *8
+immobilityIllegal = true
+flagPiece = k
+whiteFlag = *8
+blackFlag = *1
+castling = false
+
+[twokings:chess]
```

Comment:
Twokings is a variant already defined by several GUIs and engines, and the rules here differ from that, so please either rename and comment it, or remove it, in order to avoid confusion.